### PR TITLE
release: v4.1.0

### LINKED ISSUE #4803: release: 4.1.0
## Checklist

- [x] Assign to next release manager
- [ ] Close previous release milestone
- [x] Create next release milestone
- [x] Associate issue with next release milestone

### PR DESCRIPTION
**Pull request by code-asher**
Closes https://github.com/coder/code-server/issues/4803.

### CODE REVIEW

File: .github/semantic.yaml [hunk 1]
```diff
@@ -61,3 +61,6 @@ types:
   # implementations. For example, if a commit adds a fix + test, it's a fix
   # commit. If a commit is simply bumping coverage, it's a test commit.
   - test
+
+  # A new release.
+  - release
```
Comment (jsjoeio) [inline .github/semantic.yaml:66]: lol was about to check this. Nice!

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,15 +22,54 @@ VS Code v99.99.999
...
+
+- Fix web sockets not connecting when using `--cert`.
+- Prevent workspace state collisions when opening a workspace that shares the
+  same file path with another workspace on a different machine that shares the
+  same domain. This was causing files opened in one workspace to be "re-"opened
+  in the other workspace when the other workspace is opened.
+- Pin the Express version which should make installing from npm work again.
+- Propagate signals to code-server in the Docker image which means it should
+  stop more quickly and gracefully.
+- Fix missing argon binaries in the standalone releases on arm machines.
```
Comment (jsjoeio) [inline CHANGELOG.md:68]: Great changelog update!

**Review by jsjoeio — APPROVED** [review]

**Comment by jsjoeio** [comment]
Should we re-run CI?

image

**Comment by code-asher** [comment]
Yeah will rerun

**Pull request merged**